library/src/main/kotlin/com/vinted/actioncable/client/kotlin/Connection.kt

```diff
@@ -17,30 +13,24 @@ class Connection constructor(
     /**
      * Options for connection.
      *
-     * @property sslContext SSLContext
-     * @property hostnameVerifier HostnameVerifier
      * @property cookieHandler CookieHandler
      * @property query Query parameters to send on handshake.
      * @property headers HTTP Headers to send on handshake.
      * @property reconnection Whether to reconnect automatically. If reconnection is true, the client attempts to reconnect to the server when underlying connection is stale.
      * @property reconnectionMaxAttempts The maximum number of attempts to reconnect.
      * @property reconnectionDelay First delay seconds of reconnection.
      * @property reconnectionDelayMax Max delay seconds of reconnection.
-     * @property okHttpClientFactory To use your own OkHttpClient, set this option.
-     * @property okHttpWebSocketFactory To use your own WebSocket.Factory, set this option.
+     * @property webSocketFactory To use your own OkHttp WebSocket.Factory, set this option.
      */
     data class Options(
-            var sslContext: SSLContext? = null,
-            var hostnameVerifier: HostnameVerifier? = null,
```

Comment:
Actually we could keep those parameters

Reply:
@GediminasZukas, what is the point to keep those? In my opinion, they are adding the same confusion as `okHttpClientFactory`.

Reply:
Also, we could make a simple API for newcomers by providing some simple factory, which could be used as default method:
```
class SimpleFactory(val sslContext, val hostnameVerifier): WebSocket.Factory {
  ...
}
```

Reply:
> @GediminasZukas, what is the point to keep those?

I don't know exact uses cases with those. But can be that people use it. It is a legacy from original library. I would suggest to not touch those params until implementation of: https://github.com/vinted/actioncable-client-kotlin/issues/23

Reply:
I think it makes sense to put them back, because it will reduce the amount of breaking changes in this PR. More changes can come in a future update.

@GediminasZukas @neworld what do you guys want to do?

Reply:
Make sense. Let's keep it.

Reply:
@chrisfillmore , let's keep those parameters. Would be nice to implement this for now: https://github.com/vinted/actioncable-client-kotlin/pull/22#discussion_r559962495